site/content/en/docs/administration/advanced/upgrade_guide.md

```diff
@@ -53,25 +53,53 @@ To upgrade CVAT, follow these steps:
   ```shell
   docker logs cvat_server -f
   ```
+  
+## How to upgrade CVAT from v2.2.0 to v2.3.0.
 
-## How to udgrade CVAT from v1.7.0 to v2.1.0.
-
-Step by step commands how to udgrade CVAT from v1.7.0 to v2.1.0.
+Step by step commands how to upgrade CVAT from v2.2.0 to v2.3.0.
+Let's assume that you have CVAT v2.2.0 working.
+```shell
+docker exec -it cvat_db pg_dumpall > cvat.db.dump
+cd cvat
+docker-compose down
+docker volume rm cvat_cvat_db
+export CVAT_VERSION="2.3.0"
+cd ..
+mv cvat cvat_220
+wget https://github.com/opencv/cvat/archive/refs/tags/v${CVAT_VERSION}.zip
+unzip v${CVAT_VERSION}.zip && mv cvat_${CVAT_VERSION} cvat
+cd cvat
+export CVAT_HOST=cvat.example.com
+export ACME_EMAIL=example@example.com
+docker pull cvat/server:v${CVAT_VERSION}
+docker tag cvat/server:v${CVAT_VERSION} openvino/cvat_server:latest
```

Comment:
This is actual only for previous versions and doesn't matter for 2.2.0

Reply:
Not sure how it's not actual as the doc clearly says `Note: updating CVAT from version 2.2.0 to version 2.3.0 requires additional manual actions with database data due to upgrading PostgreSQL base image major version. See details [here](https://github.com/opencv/cvat/blob/develop/site/content/en/docs/administration/advanced/upgrade_guide.md#how-to-upgrade-postgresql-database-base-image)

Reply:
I mean only `docker tag ...` option